e2e-polybft/framework/test-cluster.go

```diff
@@ -250,7 +251,9 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 		}
 
 		if cluster.Config.HasBridge {
-			args = append(args, "--bridge")
+			rootchainIP, err := command.ResolveRootchainIP("")
```

Comment:
?

Reply:
Is it expeced default value there?

Reply:
Fixed [3732f6e](https://github.com/0xPolygon/polygon-edge/pull/989/commits/3732f6e0b662b6ace6f9c297999c9ff6ffc5efd1)